Visual Moderator Dashboard

Sonjajo opened the pull request:
Updated Moderator Dash with clickable apps on the left that expand to the right; added some new sample-data. 
Used the material-UI card component for the left and right cards. CSS for cardDetail on the right can still use some work, will fix this on this week's sprint when I make the form.

## Related PRs

_Optional - any related PRs you're waiting on, or PRs that will conflict, etc_

## Migrations

_Optional - if you added anything to the database through migration(s)_

## Screenshots

_Required for visual changes_
![image](https://user-images.githubusercontent.com/30053749/96393917-7cf1cf80-1175-11eb-91a0-ece32c6521c1.png)

CC: @kwfk

Sonjajo committed:
feat: debugging

Sonjajo committed:
feat: n/a

Sonjajo committed:
feat: add three components and two pages to construct the admin dash

Sonjajo committed:
refactor: rename Admin to Moderator; cleanup comments in code

Sonjajo committed:
refactor: fix the merge conflict

Sonjajo committed:
fix: fix OrgApp naming and key prop error

Sonjajo committed:
fix: add item key!

Sonjajo committed:
Merge branch 'main' of https://github.com/calblueprint/nbjc into sonja/update-mod-dash

Sonjajo committed:
feat: updated mod dash with clickable apps on the left that expand on the right

Sonjajo committed:
"update moderator dash"
Merge branch 'main' into sonja/update-mod-dash

kwfk commented on pages/moderator/index.tsx:
```diff
@@ -1,30 +1,60 @@
-    </p>
-  </Layout>
-);
+const ModList: React.FunctionComponent<Props> = ({ items }) => {
```
Let's name this ModeratorDashboard

kwfk commented on pages/moderator/index.tsx:
```diff
@@ -1,30 +1,60 @@
+        </div>
+        <div className={styles.rightCol}>
+          <div className={styles.expandedApp}>
+            {' '}
```
delete spaces

kwfk commented on pages/moderator/index.tsx:
```diff
@@ -1,30 +1,60 @@
+            <button type="button" className={styles.decline}>
+              {' '}
+              Decline
+            </button>
```
let's use filled MaterialUI buttons with primary color for accept and secondary for decline

kwfk commented on components/OrgCard.tsx:
```diff
@@ -1,24 +1,30 @@
-);
+const OrgCard: React.FunctionComponent<CardProps> = ({ items }) => {
+  return (
+    <Card className={styles.root}>
```
Add `CardActionArea` to make the card seem clickable
https://material-ui.com/components/cards/#media

kwfk replied:
You may need to fix the styling because it move the image to the top for some reason

Sonjajo replied:
done, and fixed!

kwfk commented on pages/moderator/index.tsx:
```diff
@@ -1,30 +1,60 @@
+      <h1>Moderator Dash</h1>
+      <div className={styles.root}>
+        <div className={styles.leftCol}>
+          {items.map((a) => (
```
replace `a` with something more descriptive

kwfk reviewed (changes requested):
nice work!

Sonjajo committed:
refactor: completed requested changes, including a CardActionArea, fix style bugs, improved names

Sonjajo committed:
style: clean up index buttons style

Sonjajo committed:
style: fix margins of logos on OrgCards

kwfk committed:
refactor(moderator dashboard): refactor page components into own folder for more clarity

Each component specific to a page will live inside a directory with the same name as the page
component along with the styles file. The page styles will live in the root styles directory with
the name [page-component-name].module.css.

kwfk reviewed (approved):
I made some refactoring changes so take a look at those but otherwise LGTM

kwfk commented:
actually @calblueprint/nbjc everyone should read this
